specification/components/schemas/queryParameters/RevInclude.yaml

```diff
@@ -11,7 +11,6 @@ description: |
   &category=http://snomed.info/sct|3457005
   &_revinclude=Appointment:based-on
   &_revinclude=Task:focus
-required: true
```

Comment:
How does the endpoint behave if these `_revincludes` aren't included? Is the related appointment/task included always whether or not these are included?

Reply:
they're always included